install/update.php

```diff
@@ -888,26 +888,15 @@ function pluginGlpiinventoryUpdate($current_version, $migrationname = 'Migration
     if ($crontask->getFromDBbyName('PluginGlpiinventoryTaskjob', 'updatedynamictasks')) {
         $crontask->delete($crontask->fields);
     }
-    if (!$crontask->getFromDBbyName('PluginGlpiinventoryAgent', 'cleanoldagents')) {
-        CronTask::Register(
-            'PluginGlpiinventoryAgent',
-            'cleanoldagents',
-            86400,
-            ['mode' => 2, 'allowmode' => 3, 'logs_lifetime' => 30,
-                               'hourmin' => 22, 'hourmax' => 6,
-            'comment' => Toolbox::addslashes_deep(__(
-                'Delete agents that have not contacted the server since "xxx" days.',
-                'glpiinventory'
-            ))]
-        );
+    if ($crontask->getFromDBbyName('PluginGlpiinventoryAgent', 'cleanoldagents')) {
+        $crontask->delete($crontask->fields);
```

Comment:
Why this change ?

Reply:
It's a deleted crontask as can be seen in https://github.com/glpi-project/glpi-inventory-plugin/blob/bc7d2cddeaf7641e511a5a2b3a881c54d6e5a0f4/install/update.native.php#L700, I believe it becomes native in GLPI 10 since the agent management was migrated to the core.